PROJECT

```diff
@@ -0,0 +1,8 @@
+domain: windowsmachineconfig.openshift.io
```

Comment:
Please add the command you ran to generate these files to the commit message. I am assuming rest of this commit need not be reviewed.